.all-contributorsrc

```diff
@@ -308,6 +308,16 @@
         "doc",
         "code"
        ]
+     },
+     {
+      "login": “swapnil-satpathy",
+      "name": “Swapnil Satpathy",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/54858475?s=460&u=26d2adf4592c6517f58ac87edadf4eb3bc06edba&v=4",
+      "profile": "https://github.com/swapnil-satpathy",
+      "contributions": [
+        "doc",
```

Comment:
You have not added your solution to the README markdown document, so your contribution only counts as code. Also, run the `npm run contirbutors:generate` command mentioned in the [Contribution Guidelines](https://github.com/CodeToExpress/dailycodebase/blob/master/CONTRIBUTING.md#pull-request-process-and-contribution-guidelines) so that your details are added to the CONTRIBUTORS markdown document.